base/sysinfo.jl

```diff
@@ -234,7 +234,18 @@ function loadavg()
     return loadavg_
 end
 
+"""
+    Sys.free_memory()
+
+Get the total free memory in RAM in kilobytes.
+"""
 free_memory() = ccall(:uv_get_free_memory, UInt64, ())
+
+"""
+    Sys.total_memory()
+
+Get the total memory in RAM (used and unused) in kilobytes.
```

Comment:
The `(used and unused)` comment made it feel like this returned a 2-tuple or something. Perhaps just

> Get the total memory of the RAM

or something?

Reply:
Maybe (including that which is currently used)? I want to make sure it's clear that it includes allocated memory too.

Reply:
:+1: